citi-onboarding/Recorda-one-page
# Feature/footer

What I did:

- Created footer section
- Created MediaContent component
- Added some images to 'assets' folder

How to test:

- git checkout main
- git pull
- git checkout feature/footer
- git pull
- open 'client' folder
- write "npm run start" on the client at terminal
- check footer section
- review the code
